packages/SketchMorph2-Core.package/M2Backend.class/instance/drawRectangle.st

```diff
@@ -7,4 +7,7 @@ drawRectangle
 		isErasing: false;
 		mode: #pen;
 		currentTool: #drawRectangle;
+		currentCursor: ((Form extent: self rectangleBrushSize depth: self defaultDepth)
```

Comment:
Maybe we should add a cursor for that like we did for `drawEraser` to make the code more consistent.